### ISSUE TITLE
Perfect sitemap on local build, only defaultLocale on distant build (netlify)Perfect sitemap on local build, only defaultLocale on distant build (netlify)

### ISSUE THREAD

**Issue by Joaojack** [harlan-zw/nuxt-simple-sitemap#127]
### Describe the bug

Hi guys, my issue is with i18n, nuxt simple sitemap and deployment. When running build + Generate and launching the server locally, the sitemap is perfect with all the locale paths with custom route names.
But when deploying on netlify (which creates, i guess, a build), the sitemap.xml only displays the defaultLocale pages (with correct custom routes names)
How come my local build generates the correct xml, and the distant one doesn't ?

### System / Nuxt Info

```Shell
My config :
i18n: {
baseUrl: 'https://www.foobat.com',
stratedy: 'prefix_except_default',
customRoutes: 'config',
locales: [
{
code: 'en',
iso: 'en-US',
name: 'English',
file: 'en-US.json'
},
{
code: 'fr',
iso: 'fr-FR',
name: 'Français',
file: 'fr-FR.json'
},
],
defaultLocale: 'fr',
langDir: 'lang',
pages: {
chambres: {
en: '/rooms-suites-french-riviera',
fr: '/chambres-suites-cote-dazur',
},
journal: {
fr : '/cote-d-azur-evenements-hebergement',
en : '/french-riviera-events-accommodation'
},
cgv: {
fr : '/conditions-generales-vente-hebergement-golfe-juan',
en : '/general-terms-conditions-accommodation-golfe-juan'
},
confidentialite: {
fr : '/declaration-confidentialite',
en : '/privacy-policy'
},
legal: {
fr : '/mentions-legales',
en : '/legal-notice'
},
exterieurs: {
fr : '/jardin-piscine-hebergement-golfe-juan',
en : '/garden-pool-accommodation-golfe-juan'
},
services: {
fr : '/services-luxe-cote-d-azur',
en : '/services-luxury-french-riviera'
},
informations: {
fr : '/informations-plage-hebergement-golfe-juan',
en : '/information-beach-accommodation-golfe-juan'
},
professionnels: {
fr : '/tournages-seminaires-cote-d-azur',
en : '/shootings-seminars-french-riviera'
},
}
},
sitemap: {
siteUrl: 'https://www.foobar.com'
}
```

**Comment by harlan-zw** [comment]
Hi, could you try with the latest version and let me know if the issue still persists?

**Comment by Joaojack** [comment]
Hello Harlan, works perfectly now. Thank you very much !

**Comment by Joaojack** [comment]
Problem solved with the latest release (3.3.2). Thanks !
